design/one-pager-component-mutable-and-versioning.md

```diff
@@ -0,0 +1,477 @@
+# Component mutable and versioning mechanism
+
+* Owner: Lei Zhang (@resouer), Jianbo Sun (@wonderflow)
+* Reviewers: Crossplane Maintainers
+* Status: Draft
+
+## Terminology
+
+* Application: A holistic term for talking about all pieces that are rendered from an `ApplicationConfiguration`.
+* ComponentSchematic: A term in OAM spec v1alpha1, which is deprecated and replaced by `Component` in v1alpha2.
+* ComponentRevision: A new concept introduced in this proposal which will work as revision of component.  
+
```

Comment:
Will you explain a little bit about `ControllerRevision` in this `Terminology` section?